### PR TITLE
Add: dmi. and dmi? commands

### PR DESCRIPTION
**Pull request by leberus**
```
0x7f8bc8e7c190]> dmi.
[Symbols]
vaddr=0x7f8bc907b948 paddr=0x00600948 ord=057 fwd=NONE sz=0 bind=GLOBAL type=NOTYPE name=_end

1 symbols
[0x7f8bc8e7c190]> s sym.main
[0x00400516]> dmi.
[Symbols]
vaddr=0x00400516 paddr=0x00000516 ord=060 fwd=NONE sz=11 bind=GLOBAL type=FUNC name=main

1 symbols
[0x00400516]> dmi?
|Usage: dmi  # List/Load Symbols
| dmi   List symbols of target lib
| dmi*  List symbols of target lib in radare commands
| dmi.  List closest symbol to the current address
[0x00400516]>
```

To print the symbol I'm using `r_core_bin_info` with a name-filter (the name of the closest symbol). I hope this is fine.

### CODE REVIEW

File: libr/core/cmd_debug.c [hunk 1]
```diff
@@ -885,6 +885,19 @@ static int str_start_with(const char *ptr, const char *str) {
 
 #endif // __linux__ && __GNU_LIBRARY__ && __GLIBC__ && __GLIBC_MINOR__
 
+static RDebugMap *get_closest_map(RCore *core, ut64 addr) {
+	RListIter *iter;
+	RDebugMap *map;
+
+	r_debug_map_sync (core->dbg);
+	r_list_foreach (core->dbg->maps, iter, map) {
+		if (addr != -1 && (addr >= map->addr && addr < map->addr_end)) {
```
Comment (radare) [inline libr/core/cmd_debug.c]: -1 != UT64_MAX

File: libr/core/cmd_debug.c [hunk 2]
```diff
@@ -1017,39 +1031,39 @@ static int cmd_debug_map(RCore *core, const char *input) {
...
-			}
-			r_debug_map_sync (core->dbg); // update process memory maps
-			r_list_foreach (core->dbg->maps, iter, map) {
-				if (core->bin &&
-						((addr != -1 && (addr >= map->addr && addr < map->addr_end)) ||
-						 (libname && (strstr (map->name, libname))))) {
+				i = r_str_word_set0 (ptr);
+				switch (i) {
+				case 2: // get symname
+					symname = r_str_word_get0 (ptr, 1);
```
Comment (radare) [inline libr/core/cmd_debug.c]: add a /* passthru */ comment to avoid coverity tears

**Comment by radare** [comment]
what was exactly not working before?

**Comment by leberus** [comment]
`dmi` was working for me. The only thing I've done in `dm`i code is  creating a function to find the closest map (something that `dmi.` needs too, so I've created a function to not repeat the code)
I'll address the comments.

**Comment by radare** [comment]
ill merge and test after that, pls address the comment

File: libr/core/cmd_debug.c (hunk 2 above)
Comment (leberus) [inline libr/core/cmd_debug.c]: I'll get rid of this comment, since i think it's pretty clear from the variable name

**Comment by leberus** [comment]
I've addressed the comments and squashed both commits.

**Pull request merged**